### ISSUE TITLE
gaptic: bug in ForallRightTactic

### ISSUE THREAD

**Issue by shetzl** [gapt/gapt#454]
In the gaptic-tactis branch, run the following in the CLI:
```
import at.logic.gapt.proofs.gaptic._

val drinker = new Lemma( Sequent( Nil, Seq( "D" -> parseFormula( "(exists x all y (P(x) -> P(y)))" )))) {
  use(ExistsRightTactic( parseTerm( "c" ), "D1" ))
  use(ForallRightTactic( FOLVar( "x_0" ), "D1" ))
} qed
```
This results in:
```
import at.logic.gapt.proofs.gaptic._
warning: there was one feature warning; re-run with -feature for details
Current sub goals:
0:  :- (D,∃x.∀y.(P(x)⊃P(y)))
Current sub goals:
0:  :- (D1,∀y.(P(c)⊃P(y))), (D,∃x.∀y.(P(x)⊃P(y)))
at.logic.gapt.proofs.lkNew.LKRuleCreationException: Cannot create ForallRightRule: Formula (P(c)⊃P(y)) only found 0 times in succedent of (P(c)⊃P(x_0)) :- ∃x.∀y.(P(x)⊃P(y)).
  at at.logic.gapt.proofs.lkNew.ConvenienceConstructor.LKRuleCreationException(lkNew.scala:1836)
```

Apparently the auxiliary formula is placed on the wrong side of the premise sequent.

**Comment by gebner** [comment]
cc @alexanderbirch

**Comment by alexanderbirch** [comment]
I will have a look at it. I am getting back to work on the project after my presentation later today!

**Comment by shetzl** [comment]
Fixed in 15504d169b554dd0ae7514d9babcaef7faed4847.
